api/v1/db/oracledb/player-dao.js

```diff
@@ -73,4 +73,64 @@ const getPlayerByGameIdAndPlayerId = async (id, pid) => {
   }
 };
 
-module.exports = { getPlayersByGameId, getPlayerByGameIdAndPlayerId };
+const cleanPlayerCardsByPlayerId = async (playerId) => {
+  const connection = await conn.getConnection();
+  try {
+    const sqlParams = [playerId];
+    const deleteSqlQuery = `
+    DELETE FROM PLAYER_CARDS WHERE PLAYER_ID = :playerId
+    `;
+    const result = await connection.execute(deleteSqlQuery, sqlParams, { autoCommit: true });
+    return result;
+  } finally {
+    connection.close();
+  }
+};
+
+const deletePlayerByPlayerId = async (playerId) => {
+  const connection = await conn.getConnection();
+  try {
+    await cleanPlayerCardsByPlayerId(playerId);
+    const sqlParams = [playerId];
+    const deleteSqlQuery = `
+    DELETE FROM PLAYERS WHERE PLAYER_ID = :playerId
+    `;
+    const result = await connection.execute(deleteSqlQuery, sqlParams, { autoCommit: true });
+    return result;
+  } finally {
+    connection.close();
+  }
+};
+
+
+const deletePlayersByGameId = async (gameId) => {
+  const connection = await conn.getConnection();
+  try {
+    const sqlParams = [gameId];
+    const playerSqlQuery = `
+    SELECT PLAYER_ID FROM PLAYERS P
+    WHERE GAME_ID = :gameId
+    `;
+    const rawPlayersResponse = await connection.execute(playerSqlQuery, sqlParams);
+    const playerIds = _.map(rawPlayersResponse.rows, player => (player.PLAYER_ID));
+    if (!_.isEmpty(playerIds)) {
+      const deletePlayerCardSqlQuery = `
+      DELETE FROM PLAYER_CARDS WHERE PLAYER_ID IN (${playerIds.map((name, index) => `:${index}`).join(', ')})`;
+      await connection.execute(deletePlayerCardSqlQuery, playerIds, { autoCommit: true });
+
+      const deletePlayersSqlQuery = `
+      DELETE FROM PLAYERS WHERE PLAYER_ID IN (${playerIds.map((name, index) => `:${index}`).join(', ')})`;
```

Comment:
Since `playerIds.map((name, index) => ...` is repeated here, could you store it in a variable?